lib/runtime/interfaces.go

```diff
@@ -11,28 +11,34 @@ import (
 
 // Storage runtime interface.
 type Storage interface {
+	// Main trie
```

Comment:
Nice, I am a fan of labeling and sorting methods in large interfaces like this

Reply:
Yes, this is one approach. Another approach involves creating a sort of 'inheritance' between interfaces. We can create four interfaces, each for a different group, and then compose the `Storage` interface from those other interfaces.